.github/workflows/continous-integration.yml

```
@@ -295,7 +295,15 @@ jobs:
       run: pip install pytest-github-actions-annotate-failures
 
     - name: Test Code 🔍
-      if: needs.changes.outputs.backend == 'true' && !(matrix.python-version == 3.8 && matrix.os == 'windows-latest')
+      if: needs.changes.outputs.backend == 'true' && !(matrix.python-version == 3.8 && matrix.os == 'windows-latest') && (matrix.os == 'windows-latest' && matrix.test == 'test-nlu-predictors')
+      env:
+        JOBS: 2
+        PYTHONIOENCODING: "utf-8"
+      run: |
+        make ${{ matrix.test }}
+
+    - name: Test Code 🔍
+      if: needs.changes.outputs.backend == 'true' && !(matrix.python-version == 3.8 && matrix.os == 'windows-latest') && !(matrix.os == 'windows-latest' && matrix.test == 'test-nlu-predictors')
```

```suggestion
      if: needs.changes.outputs.backend == 'true' && (matrix.python-version == 3.8 && matrix.os == 'windows-latest') && !(matrix.os == 'windows-latest' && matrix.test == 'test-nlu-predictors')
```
I believe you want the opposite condition here no?

---

I'm unsure, I was trying to keep the original behaviour: I copied this bit over from the original [step](https://github.com/RasaHQ/rasa/blob/2.8.x/.github/workflows/continous-integration.yml#L298) and put it into both conditions because I thought we don't want this config ever to run? Or is that not the case anymore?

`!(matrix.python-version == 3.8 && matrix.os == 'windows-latest')`

---

oh sorry, I missed that -- seems we're already not running win + py3.8 at all, indeed. Nevermind me, my bad ahah